page.go

```diff
@@ -15,19 +16,21 @@ import (
 type page struct {
 	// offset is the offset in the file that this page has.
 	offset uint64 // Is NOT marshalled to disk.
-
```

Comment:
should add back in this newline. pageStatus is so heavily commented that it really deserves its own block